controllers/loadtest_controller.go

```diff
@@ -139,12 +163,36 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 	}
 
+	reQueueTime := getRequeueTime(previousStatus, test, log)
```

Comment:
nit: let's keep the same case as the function. 

```suggestion
	requeueTime := getRequeueTime(previousStatus, test, log)
```